docs/reference/stable-patch-format.rst

```diff
@@ -27,46 +27,51 @@ Every patch submitted to a stable kernel **must** have have its subject line
 starting with "[SRU]" followed by the release name against which the
 patch is targeted.
 
-The release name **must** be enclosed in "[ ]" brackets and can be
-abbreviated using the first letter of the release name (e.g. "B" for
-"Bionic"). For example:
+The release name **must** be enclosed in "[ ]" brackets and **should** be
+abbreviated using the first letter of the release name (e.g. "N" for
+"Noble"). For example:
 
 .. code-block:: none
 
-   [SRU][Focal][PULL] Focal update: v5.4.56 upstream stable release
+   [SRU][N][PATCH 0/1] Fix error of resume on rtl8168fp
 
 * If a patch is to be applied to multiple releases, a list of release names
-  must be provided. For example:
+  must be provided. For example, when it targets Bionic and Focal:
 
   .. code-block:: none
 
-     [SRU][B][F][PATCH 1/1] KVM: fix overflow of zero page refcount with ksm running
+     [SRU][B/F][PATCH 1/1] KVM: fix overflow of zero page refcount with ksm running
 
-* If the patch has to be applied to multiple releases, you can drop some of
-  the square brackets and use the comma "," or forward slash "/" character
-  to separate the releases.
+  .. note::
 
-  For example, if the patch has to be applied to the "Bionic, "Focal", and
-  "Jammy" release, change from this:
+     Historically, the rule has been somewhat flexible, and various styles
+     have been permitted. You may find examples like the following in the
+     `mailing list archive
+     <https://lists.ubuntu.com/archives/kernel-team/>`__:
 
-  .. code-block:: none
-
-     [SRU][B][F][J][PATCH 0/1] UBUNTU: [Config]: Add support for modifying LDT
-
-  To a comma- or forward slash-separated list:
+     .. code-block:: none
 
-  .. code-block:: none
+        [SRU][B,F][PATCH 1/1] KVM: fix overflow of zero page refcount with ksm running
+        or
```

Comment:
I don't think it's a good practice to mix in the "OR"s with the examples you're trying to demonstrate
I suggest adding line numbers instead to help with the description before the code block.